lab_3_ann_retriever/main.py

```diff
@@ -49,6 +52,12 @@ def calculate_distance(query_vector: Vector, document_vector: Vector) -> float |
 
     In case of corrupt input arguments, None is returned.
     """
+    if len(query_vector) != len(document_vector):
```

Comment:
try to think more about real tests. A lot of tests in our labs are type checkers, but also you should think about real testcases. For example what will be if vector will be like: `[0, -100, -212, 1]` ?

Reply:
if not all(i >= 0 for i in query_vector) or not all(k >= 0 for k in document_vector) ?

Reply:
yes may be